tests_config/run_spec.js

```diff
@@ -63,14 +63,15 @@ function run_spec(dirname, parsers, options) {
       if (AST_COMPARE) {
         const compareOptions = Object.assign({}, mergedOptions);
         delete compareOptions.cursorOffset;
-        const astMassaged = parse(input, compareOptions);
+        const astMassaged =
+          input && input.trim().length ? parse(input, compareOptions) : {};
```

Comment:
This might no be the correct fix for this either... maybe we should handle that in `prettier.__debug.parse`

Reply:
I reverted this as it got fixed when I rebased with master.